src/app/components/navbar/navbar.component.html

```diff
@@ -22,16 +22,16 @@
       >
         <ul class="navbar-nav ml-auto">
           <li class="nav-item">
-            <a class="nav-link" data-toggle="collapse" data-target="#navbarCollapse" routerLink="/home">Home</a>
+            <a class="nav-link" (click)="toggleCollapsed()" routerLink="/home">Home</a>
```

Comment:
It is not enough to toggle the collapsed state alone. Now, only the styling changes and the navbar doesn't collapse or show correctly anymore